src/Exadel.CrazyPrice/Exadel.CrazyPrice.WebApi/Extentions/ControllerExtentions.cs

```diff
@@ -11,5 +11,9 @@ public static Guid GetUserId(this ControllerContext controllerContext)
         {
             return controllerContext.HttpContext.User.Claims.Where(c => c.Type == "sub").Select(k => k.Value).FirstOrDefault().ToGuid(Guid.Empty);
         }
+        public static string GetRole(this ControllerContext controllerContext)
+        {
+            return controllerContext.HttpContext.User.Claims.Where(c => c.Type == "role").Select(k => k.Value).FirstOrDefault();

```

Comment:
What happens if at least User or Claims is null or Empty?

Reply:
NullReferenceExeption. Fixed is!